Dockerfile

```diff
@@ -28,8 +30,12 @@ RUN cp \
     /usr/local/bin/psql \
     /usr/local/lib/libpq.so.5 \
     /usr/local/lib/libpq.so.5.16 \
+    /usr/lib/libldap.so.2 \
+    /lib/libc.musl-x86_64.so.1 \
     ./bin
- 
+
+RUN chmod +x bin/libldap.so.2
```

Comment:
not totally necessary, I just did this to match all the other files in `bin` to hopefully prevent needing a follow up deployment